<launch>
    <!-- Move Base -->
    <node name="move_base" pkg="move_base" type="move_base" respawn="false" output="screen">
        <param name="base_global_planner" value="global_planner/GlobalPlanner" />
        <param name="base_local_planner" value="dwa_local_planner/DWAPlannerROS"/>
        <!-- <param name="base_local_planner" value="teb_local_planner/TebLocalPlannerROS"/> -->
        <rosparam file="$(find netrasahaya)/config/costmap_common_params.yaml" command="load" ns="global_costmap" />
        <rosparam file="$(find netrasahaya)/config/costmap_common_params.yaml" command="load" ns="local_costmap" />
        <rosparam file="$(find netrasahaya)/config/move_base_params.yaml" command="load" />
        <rosparam file="$(find netrasahaya)/config/local_costmap_params.yaml" command="load" />
        <rosparam file="$(find netrasahaya)/config/global_costmap_params.yaml" command="load" />
        <rosparam file="$(find netrasahaya)/config/base_local_planner_params.yaml" command="load" />
    </node>
</launch>